ga4gh/frontend.py

```diff
@@ -111,11 +149,16 @@ def handleException(exception):
     return response
 
 
-def handleFlaskRequest(flaskRequest, endpoint):
+def handleFlaskPostRequest(version, flaskRequest, endpoint):
+    if Version.parseString(version) != Version.parseString(protocol.version):
```

Comment:
Because `version` and `protocol.version` are different strings.  Specifically, `version` has a v in front of it.  This is considered a best practice for a url version string.